install.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
```

Comment:
Could we leave this in please? If there's an error things scroll past and one might very well not notice that the install failed, and then one spends ages chasing ghosts. Speaking from personal experience.